src/views/StartView.vue

```diff
@@ -1,69 +1,163 @@
+        </h2>
       </v-col>
     </v-row>
-
-    <h2 class="text-h4">{{ $t('start.administration') }}</h2>
-    <v-row>
-      <v-col
-        cols="12"
-        sm="6"
-        md="4"
-      >
-        <ServiceProviderCard
-          :testId="'service-provider-card-admin'"
-          :to="'/admin/personen'"
-          :title="$t('nav.admin')"
-          variant="outlined"
+    <v-divider
+      class="border-opacity-100 rounded"
+      color="#1EAE9C"
+      thickness="5px"
+    ></v-divider>
+    <v-row class="flex-nowrap my-3">
+      <v-col cols="auto">
+        <h2
+          class="headline-2"
+          data-testid="all-service-provider-title"
         >
-        </ServiceProviderCard>
+          {{ $t('start.allServiceProviders') }}
+        </h2>
       </v-col>
     </v-row>
-  </div>
+    <!-- Template to be displayed in case the providers are still being loaded -->
+    <template v-if="serviceProviderStore.loading">
+      <v-progress-circular indeterminate></v-progress-circular>
+    </template>
+    <!-- Template to be displayed in case loading the providers throws an error -->
+    <template v-else-if="serviceProviderStore.errorCode">
+      <p
+        class="text-caption text-left"
+        data-testid="error-text"
+      >
+        {{ $t(`errors.${serviceProviderStore.errorCode}`) }}
+      </p>
+    </template>
+    <!-- Template to be displayed in case of no error nor loading -->
+    <template v-else>
+      <!-- Categorie 1: Work Email -->
+      <ServiceProviderCategory
+        :categoryTitle="$t('start.categories.workEmail')"
+        :serviceProviders="emailServiceProviders"
+      ></ServiceProviderCategory>
+      <!-- Categorie 2: Class -->
+      <ServiceProviderCategory
+        :categoryTitle="$t('start.categories.class')"
+        :serviceProviders="classServiceProviders"
+      ></ServiceProviderCategory>
+      <!-- Categorie 3: Administration directly rendered here because of the always present provider -->
+      <v-row>
+        <label class="mx-3">{{ $t('start.categories.administration') }}</label>
+        <v-col>
+          <v-divider
+            class="border-opacity-100 rounded"
+            color="#E5EAEF"
+            thickness="6"
+          ></v-divider>
+        </v-col>
+      </v-row>
+      <v-row>
+        <v-col
+          v-for="serviceProvider in administrationServiceProviders"
+          :key="serviceProvider.id"
+          cols="12"
+          md="6"
+          lg="4"
+        >
+          <ServiceProviderCard
+            :href="serviceProvider.url"
+            :newTab="true"
+            :testId="`service-provider-card-${serviceProvider.id}`"
+            :title="serviceProvider.name"
+            :logoUrl="serviceProvider.logoUrl"
+          ></ServiceProviderCard>
+        </v-col>
+        <v-col
+          cols="12"
+          md="6"
+          lg="4"
+        >
+          <ServiceProviderCard
+            :testId="'service-provider-card-admin'"
```

Comment:
```suggestion
            testId="service-provider-card-admin"
```
Einfacher String sollte hier ausreichen